README.md

```diff
@@ -0,0 +1,144 @@
+# Performance Characterization
+
+## Test Cases
+
+Test Case | Description | Notes
+---------|----------|---------
+ 1 | FSPIOP GetParties End-to-end | FSPIOP GET /parties request executed from K6 via the Account-Lookup-Service
+
+## Test Scenarios
+
+Scenario | Description | Test-Case | Repeatable (Y/N) | Notes
```

Comment:
Can this table be updated with the changes in the scenario descriptions in the respective folders